backend/src/apiserver/resource/resource_manager_test.go

```diff
@@ -4020,6 +4103,33 @@ func TestCreateTask(t *testing.T) {
 	assert.Equal(t, expectedTask, storedTask, "The StoredTask return has unexpected value")
 }
 
+func TestGetArtifactSessionInfo(t *testing.T) {
```

Comment:
the name here is a bit too vague, can we name it something like: 

`TestSupportForMultipleSessionInfo` OR
`TestBackwardsCompatibilityForSessionInfo` so the future developer understands what we are testing for here